.github/workflows/ci.yml

```diff
@@ -392,6 +392,7 @@ jobs:
       - run: sudo apt-get -y install libsoapysdr-dev
       - run: cargo test --all-targets --workspace --features=aaronia_http,rtlsdr,zeromq,audio,flow_scheduler,tpb_scheduler,seify_dummy,soapy,lttng,zynq,wgpu
       - run: cargo test --all-targets --manifest-path=crates/futuredsp/Cargo.toml
+      - run: cargo test --all-targets --all-features --manifest-path=crates/types/Cargo.toml
```

Comment:
Not sure I understand why sub-crate tests don't get run automatically, but this seems to be required to get test in `futuresdr-types` to run?

Reply:
I never questioned it :-) but I just had a look and there seems to be a workspace flag for tests. However, since only few crates are actually part of the same workspace, I'd say we just do it like this for all crates.